modules/core/client/directives/page-title.client.directive.js

```diff
@@ -19,13 +19,17 @@
 
       function listener(event, toState) {
         var applicationCoreTitle = 'MEAN.js',
-          separeteBy = ' - ';
+          separator = ' - ',
+          stateTitle = applicationCoreTitle + separator;
+
+        toState.name.split('.').forEach(function(value, index) {
+          stateTitle = stateTitle + value.charAt(0).toUpperCase() + value.slice(1) + separator;
+        });
         if (toState.data && toState.data.pageTitle) {
-          var stateTitle = $interpolate(toState.data.pageTitle)($state.$current.locals.globals);
-          element.html(applicationCoreTitle + separeteBy + stateTitle);
-        } else {
-          element.html(applicationCoreTitle);
+          stateTitle = $interpolate(stateTitle + toState.data.pageTitle + separator)(($state.$current.locals.globals));
         }
+        stateTitle = stateTitle.slice(0, 0 - separator.length);
+        element.html(stateTitle);
```

Comment:
Would just `element.text()` be enough here? I feel `.html()` being a bit risky for security.